package/yast2-network.changes

```diff
@@ -1,3 +1,13 @@
+-------------------------------------------------------------------
+Thu Nov  4 15:02:24 UTC 2021 - Knut Anderssen <kanderssen@suse.com>
+
+- AutoYaST
+  - When the interface section contains the deprecated device
```

Comment:
I would use quotes to make the text easier to understand. Moreover, I would rephrase it like *contains the "device" (deprecated) and "name" elements...* or *contains the "device" (deprecated) in addition to the "name" element...".